src/main/webapp/resources/celJS/initCelements.js

```diff
@@ -568,29 +567,29 @@
   }
 
   window.CELEMENTS.UrlUtils = Class.create({
-    getParams : function(search) {
-      var _me = this;
+    getParams: function (search) {
+      const _me = this;
       search = search || window.location.search;
-      var paramSplit = search.split(new RegExp('[?&]'))
+      let paramSplit = search.split(new RegExp('[?&]'))
```

Comment:
`paramSplit` is never reassigned
```suggestion
      const paramSplit = search.split(new RegExp('[?&]'))
```